src/zero/jdbc.clj

```diff
@@ -3,80 +3,90 @@
                            (jdbc/execute! ~db ~sql-params ~opts))))
 
 (defmacro db-do-commands
   "Logged alias for [[clojure.java.jdbc/db-do-commands]]"
   ([db sql-commands] `(do
-                        (log/info "JDBC SQL db-do-commands:" (clean-db ~db) ~sql-commands)
+                        (log/info "JDBC SQL db-do-commands:" (format-db ~db) ~sql-commands)
                         (jdbc/db-do-commands ~db ~sql-commands)))
   ([db transaction? sql-commands] `(do
-                                     (log/info "JDBC SQL db-do-commands:" (clean-db ~db) ~transaction? ~sql-commands)
+                                     (log/info "JDBC SQL db-do-commands:" (format-db ~db) ~transaction? ~sql-commands)
                                      (jdbc/db-do-commands ~db ~transaction? ~sql-commands))))
 
 (defmacro insert!
   "Logged alias for [[clojure.java.jdbc/insert!]]"
   ([db table row] `(do
-                      (log/info "JDBC SQL insert-rows! (without opts):" (clean-db ~db) ~table [~row])
-                      (jdbc/insert! ~db ~table ~row)))
+                     (log/info "JDBC SQL insert-rows! (without opts):" (format-db ~db) ~table [~row])
+                     (jdbc/insert! ~db ~table ~row)))
   ([db table cols-or-row values-or-opts] `(do
-                                             (if (map? values-or-opts)
-                                               (log/info "JDBC SQL insert-rows!:" (clean-db ~db) ~table [~cols-or-row] ~values-or-opts)
-                                               (log/info "JDBC SQL insert-cols! (without opts):" (clean-db ~db) ~table ~cols-or-row [~values-or-opts]))
-                                             (jdbc/insert! ~db ~table ~cols-or-row ~values-or-opts)))
+                                            (if (map? values-or-opts)
+                                              (log/info "JDBC SQL insert-rows!:" (format-db ~db) ~table [~cols-or-row] ~values-or-opts)
+                                              (log/info "JDBC SQL insert-cols! (without opts):" (format-db ~db) ~table ~cols-or-row [~values-or-opts]))
+                                            (jdbc/insert! ~db ~table ~cols-or-row ~values-or-opts)))
   ([db table cols values opts] `(do
-                                  (log/info "JDBC SQL insert-cols!:" (clean-db ~db) ~table ~cols [~values] ~opts)
+                                  (log/info "JDBC SQL insert-cols!:" (format-db ~db) ~table ~cols [~values] ~opts)
                                   (jdbc/insert! ~db ~table ~cols ~values ~opts))))
 
 (defmacro with-db-transaction
   "Logged alias for [[clojure.java.jdbc/with-db-transaction]]"
   [binding & body]
   `(let [id# (rand-int 10000)]
-     (log/info "JDBC SQL transaction" id# "started to" (clean-db ~(second binding)))
+     (log/info "JDBC SQL transaction" id# "started to" (format-db ~(second binding)))
      (jdbc/with-db-transaction ~binding ~@body)
      (log/info "JDBC SQL transaction" id# "ended")))
```

Comment:
See #78

Reply:
Merged your branch into this one to apply the fix